helpers/installcredprovider.ps1

```diff
@@ -111,13 +113,20 @@ function InstallZip {
         $releaseJson = $release.Content | ConvertFrom-Json
```

Comment:
Doesn't the null check for $releaseJson need to be before we use the variable in line 114? Would line 114 throw without ever getting to the null check below?

Reply:
Same with the null check for $zipAsset

Reply:
And while we're at it, might as well add a null check for $release as well?

Reply:
From my understanding and testing, having a null value for $releaseJon, $zipAsset, or even $packageSourceUrl wont break the script. The following function calls (ConvertFrom-Json / where-object ) will just be made on a null value, resulting in another null return value. So the changes your suggesting would save us a few function calls, but may reduce readability. I do still think that implementing them is a good idea though, since faster run time is never a bad thing. 

Also, I'm not sure if a null check for $release is needed. the Invoke-WebRequest function in line 112 should either be successful and return a WebResponseObject, or throw an error. If I am misunderstanding this or if it will still be beneficial to have, please let me know!

Reply:
I think it is okay to leave the null check there because if the Invoke-WebRequest does change under the hood or in new version, it would be "safer"